api/openshift.deploy.yml

```diff
@@ -290,6 +290,8 @@ objects:
         app: ${NAME}-${ZONE}
         certbot-managed: ${CERTBOT}
       name: ${NAME}-${ZONE}-${COMPONENT}
+      annotations:
+        haproxy.router.openshift.io/timeout: 90s # (Fix 611 - /bcgw-extract long time response in prod)
```

Comment:
@basilv , would you prefer for a longer value here?

Reply:
90s is only double the current query duration. That might be exceeded in the next month. Lets go to 120s.

Reply:
Increased to 120 now.